drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/ChromeDriverBinaryHandler.java

```diff
@@ -87,6 +88,25 @@ protected String getExpectedKeyRegex(String requiredVersion, String directory) {
             return getExpectedKeyRegex(requiredVersion, directory, Architecture.AUTO_DETECT);
         }
 
+        @Override
+        public ExternalBinary getReleaseForVersion(String requiredVersion, Architecture architecture) throws Exception {
+            ExternalBinary release;
+            try {
+                release = super.getReleaseForVersion(requiredVersion, architecture);
+            } catch (MissingBinaryException mbe) {
+                if (PlatformUtils.isWindows() && Objects.equals(architecture.getValue(), Architecture.BIT64.getValue())) {
```

Comment:
Can't we say `Architecture.BIT64 == architecture` for enums?

Reply:
Yes, good point. That was my first assumption too. Unfortunately we have a third constant `Architecture.AUTO_DETECT` that is dynamically evaluated and may point as to `32` so to `64` bit version.

This third constant is used for example in `XmlStorageSource`
```
    @Override
    public ExternalBinary getReleaseForVersion(String requiredVersion) throws Exception {
        return getReleaseForVersion(requiredVersion, Architecture.AUTO_DETECT);
    }
```

In other words, `Architecture.BIT64 == architecture` is evalueated to false on `64` machine, if `AUTO_DETECT` was used.

Reply:
Ah right!

Reply:
@jomarko I just realized my comment could have been perceived as rude which was not my intention at all. Apologies.

Reply:
@bartoszmajsak not at all. No need to apologize. Do not worry!